composer.json

```diff
@@ -6,7 +6,7 @@
     "require": {
         "drupal/search_api": "^1.11",
         "drupal/elasticsearch_connector": "^5.0-alpha3",
-        "dpc-sdp/tide_core": "^1.2.5"
+        "dpc-sdp/tide_core": "dev-feature/SDPA-2987_term_slug_search as 1.2.6"
```

Comment:
TODO: to remove when https://github.com/dpc-sdp/tide_core/pull/83 is merged and released.